scripts/run-solidity-tests.sh

```diff
@@ -22,9 +20,7 @@ else
 fi
 
 chmod +x ./init-test-node.sh
-./init-test-node.sh > ethermintd.log &
-sleep 5
-ethermintcli rest-server --laddr "tcp://localhost:8545" --unlock-key localkey,user1,user2 --chain-id $CHAINID --trace --wsport 8546 > ethermintcli.log &
```

Comment:
does the rest-server get automatically get deployed now with the new binary update? Or do we need to put back this command with the proper binary name?

also looks like `./init-test-node.sh` has references to `ethermintcli` which needs to be removed